shell/platform/darwin/ios/framework/Headers/FlutterViewController.h

```diff
@@ -47,8 +47,13 @@ extern NSNotificationName const FlutterSemanticsUpdateNotification;
  * Dart-related state and asynchronous tasks when navigating back and forth between a
  * FlutterViewController and other `UIViewController`s.
  */
+#ifdef __IPHONE_13_4
```

Comment:
This is conditionally removing the FLUTTER_EXPORT.  If we want to go this route you should make the scope the the conditional compilation around the UIPointerInteractionDelegate.

Reply:
good catch, I'll fix that. We should be FLUTTER_EXPORTING in every case. My bad.